modules/ROOT/pages/aurads/tutorials/arrow-examples.adoc

```diff
@@ -97,36 +93,41 @@ gds.graph.list()
 
 == Streaming node and relationship properties
 
-https://neo4j.com/docs/graph-data-science-client/current/graph-object/#graph-object-streaming-properties
+After creating the graph, you can read the node and relationship properties link:https://neo4j.com/docs/graph-data-science-client/current/graph-object/#graph-object-streaming-properties[as streams].
+These procedures do not require Arrow, but are more efficient if Arrow is available.
```

Comment:
This sentence is also valid for the `Loading data` section.
Maybe it fits better in the intro?

Reply:
True, I'll rephrase it.